tests/FunctionalTestCase.php

```diff
@@ -64,6 +68,10 @@ public function setUp(): void
 
     public function tearDown(): void
     {
+        if (! $this->hasFailed()) {
+            $this->cleanupCollections();
```

Comment:
Overriding the `tearDown` method in child classes is no longer necessary: the collection will be automagically dropped when the test calls `createCollection` or `dropCollection`.